layout/TopNav.tsx

```diff
@@ -37,16 +37,13 @@ export const NAV_ITEMS: NavItem[][] = [
   ],
 ];
 
-if (getSiteVersion() === SITE_VERSION_HACKATHON_2023) {
+if (getSiteVersion() === SITE_VERSION_PROGRAM_2023) {
   NAV_ITEMS[0].push({
-    id: "hackathon-2023",
-    label: "2023 Hackathon",
-    route: "/hackathon-2023",
+    id: "program-2023",
+    label: "Program Fall 2023",
+    route: "/program-2023",
```

Comment:
Appreciate renaming the `hackathon` components since we're no longer using the hackathon branding!